jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/LambdaHandlerResolver.kt

```diff
@@ -37,5 +37,14 @@ interface LambdaHandlerResolver {
      */
     fun determineHandler(element: PsiElement): String?
 
+    /**
+     * Return a set of valid handlers from the given [PsiElement]. Different from [determineHandler], it returns all the valid
+     * handler strings contained by the [element].
+     *
+     * For Java implementation, if the element is a Java class, it returns all the valid methods either defined by this class or its
+     * super class(es).
+     */
+    fun determineHandlers(element: PsiElement): Set<String>
```

Comment:
I don't understand how a single `PsiElement` could have more than one handler associated with it. We do the determination based on either a "Class" or a "method" element (or an identifier with a class or method as it's parent) a class's handler will always be the fully-qualified class name and a method will be the fully-qualified class name with "::[methodName]" appended.